src/test/java/bio/terra/app/controller/DatasetsApiControllerTest.java

```diff
@@ -289,4 +321,12 @@ private void verifyAuthorizationsCall(List<IamAction> iamActions) {
         .verifyAuthorizations(
             TEST_USER, IamResourceType.DATASET, DATASET_ID.toString(), iamActions);
   }
+
+  private void mockValidators() {
```

Comment:
Do you need to do this for any other tests? I am surprised that you needed to add this, and there wasn't an existing approach to these.

Reply:
Yeah its weird! just this and the other test in this file that does a post (testUpdateSnapshotBuilderSettings)

Reply:
And also in SnapshotValidationTest

Reply:
Right, yes! I think we ran into this before, I just couldn't remember the conversation around it.